From 49eb190518c5000d48c929b21304edd2e8522ea6 Mon Sep 17 00:00:00 2001
From: Steve French <stfrench@microsoft.com>
Origin: Upstream, https://git.samba.org/?p=cifs-utils.git;a=commit;h=49eb190518c5000d48c929b21304edd2e8522ea6
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1886551
Date: Sat, 16 Mar 2019 15:42:40 -0500
Subject: [PATCH] smbinfo: missing help for fsctl-getobjid

Add usage description for new option fsctl-getobjid

See section 2.1.3.1 of MS-FSCC

Signed-off-by: Steve French <stfrench@microsoft.com>
---
 smbinfo.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/smbinfo.c b/smbinfo.c
index 268114b..33fca95 100644
--- a/smbinfo.c
+++ b/smbinfo.c
@@ -88,7 +88,9 @@ usage(char *name)
 		"  secdesc:\n"
 		"      Prints the security descriptor for a cifs file.\n"
 		"  quota:\n"
-		"      Prints the quota for a cifs file.\n",
+		"      Prints the quota for a cifs file.\n"
+		"  fsctl-getobjid:\n"
+		"      Prints the objectid of the file and GUID of the underlying volume.\n",
 		name);
 	exit(1);
 }
-- 
2.17.1

